### DISCUSSION TITLE
"Package has no specified license", but does.

### DISCUSSION THREAD

**Discussion by theopenwebjp** [sandworm-hq/sandworm-audit#121] [Q&A] [answered]
https://sandworm.dev/npm/package/@theopenweb/webpack-helpers
In "critical severity issues", there are some examples such as "terser@5.19.2" stating "Package has no specified license", but clearly have one on the GitHub page:
https://github.com/terser/terser/blob/8f220355859aa0dfa49cce2ed8ac2c3bf907f8a5/package.json
What is the issue?

**Comment by gabidobo** [comment, marked as answer]
Hi @theopenwebjp, thanks for submitting this!
We had an issue with our package parsing queue and were missing info on a number of packages. Should be fixed now 👍
